### ISSUE TITLE
Abnormal neuron physiology

### ISSUE THREAD

**Issue by sarahstofel** [obophenotype/human-phenotype-ontology#7053]
**Preferred term label:**

Abnormal neuron physiology

**Synonyms**

**Definition (free text, please give PubMed ID)**

Any functional anomaly of the cells of the nervous system that receive, conduct, and transmit impulses. 
PMID:9130294

**Parent term (use hpo.jax.org/app)**

abnormal nervous system physiology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

OMIM: 104300

**Your nano-attribution (ORCID)**

https://orcid.org/0000-0003-2307-1226

**Comment by pnrobinson** [comment]
In human medicine it does not seem typical to measure abnormalities at the level of the individual neuron, and thus I am uncertain what specific abnormalities we would put underneath this term. If we only have this term, it is essentially an inference about abnormal physiology rather than a phenotypic abnormality and thus is out of scope for HPO.
See https://github.com/obophenotype/human-phenotype-ontology/issues/7055

I would suggest if needed to discuss terms like this and make additions it would be good to set up a zoom conference.
